lib/git_worktree.rb

```diff
@@ -25,6 +25,18 @@ def initialize(options = {})
     @remote_name = 'origin'
     @base_name   = File.basename(@path)
 
+    # Requires ensure at gem installation time that libssh2 is available
```

Comment:
"Requires ensure"?

Reply:
I blame it on late night coding... will fix.